### PR TITLE
♿ Add aria-label to Pinterest save button

### PR DESCRIPTION
**Pull request by andrewb**
This adds an `aria-label` to the Pinterest save button if text content is not present, i.e. the button is round.

This fixes the "links do not have a discernible name" issue flagged by Lighthouse.

Screen Shot 2020-06-11 at 8 41 53 AM

cc @caroqliu and @cvializ

### CODE REVIEW

**Comment by andrewb** [comment]
@googlebot I signed it!

File: extensions/amp-pinterest/0.1/test/test-amp-pinterest.js [hunk 1]
```diff
@@ -44,8 +51,8 @@ describes.realWin(
       pin.setAttribute('data-media', pinMedia);
       pin.setAttribute('data-description', pinDescription);
       pin.setAttribute('layout', 'responsive');
-      pin.setAttribute('width', '100');
-      pin.setAttribute('height', '100');
+      pin.setAttribute('width', width);
```
Comment (kristoferbaxter) [inline extensions/amp-pinterest/0.1/test/test-amp-pinterest.js:54]: This seems to be unrelated to the `aria-label`, do you want to include this in the PR?

If so, we should likely add some tests for the default values.

File: extensions/amp-pinterest/0.1/test/test-amp-pinterest.js [hunk 2]
```diff
@@ -34,7 +34,14 @@ describes.realWin(
       );
     }
 
-    function getPin(pinDo, pinUrl, pinMedia, pinDescription) {
+    function getPin(
+      pinDo,
+      pinUrl,
+      pinMedia,
+      pinDescription,
+      width = 20,
```
Comment (wassgha) [inline extensions/amp-pinterest/0.1/test/test-amp-pinterest.js:42]: Any reason for this change from 100 ?

Comment (andrewb) [reply extensions/amp-pinterest/0.1/test/test-amp-pinterest.js:42]: Thanks for the quick review @wassgha and @kristoferbaxter. I modified the default width and height values in the tests so `this.round` would not default to `true` (and so that the specs could override the dimensions).
 https://github.com/ampproject/amphtml/blob/a4df04761e3e3b69b51f11f8b82c6eeabbd53c6a/extensions/amp-pinterest/0.1/save-button.js#L62-L67.

Comment (wassgha) [reply extensions/amp-pinterest/0.1/test/test-amp-pinterest.js:42]: I see! Sounds good! Thank you :)
